### ISSUE TITLE
`No module named 'scripts.iib'` after updated to the lasted version.

### ISSUE THREAD

**Issue by 925-Studio** [zanllp/sd-webui-infinite-image-browsing#722]
I used `git pull` to update IIB like I always did, but this time it stops working. Error message shows in terminal:

```
Traceback (most recent call last):
  File "/home/silver/Playgrounds/AI/infinite-image-browsing/app.py", line 7, in <module>
    from scripts.iib.api import infinite_image_browsing_api, index_html_path, DEFAULT_BASE
ModuleNotFoundError: No module named 'scripts.iib'
```

Any idea what happened? Thank you for your time.

**Comment by 925-Studio** [comment]
> Did you start the wrong script? It should be `python app.py`.

I'm not sure. I created a shell file to start IIB couple months ago and it worked fine until today.

```
#!/bin/bash

cd infinite-image-browsing

python3 app.py --host="0.0.0.0" --port=4000
```

Any ideas?

**Comment by zanllp** [comment]
An unbelievable issue, I can't reproduce it. Could you provide more information or try running it with Python directly instead of through a shell script?

![image](https://github.com/user-attachments/assets/fdaf222c-f10b-4ef7-8593-6b11a7e29e32)

**Comment by 925-Studio** [comment]
> An unbelievable issue, I can't reproduce it. Could you provide more information or try running it with Python directly instead of through a shell script?
> 

I did, still got the same problem.

```
silver@MustangShelby:~/Playgrounds/AI/infinite-image-browsing$ python3 app.py
Traceback (most recent call last):
  File "/home/silver/Playgrounds/AI/infinite-image-browsing/app.py", line 7, in <module>
    from scripts.iib.api import infinite_image_browsing_api, index_html_path, DEFAULT_BASE
ModuleNotFoundError: No module named 'scripts.iib'
```

**Comment by zanllp** [comment]
I can only guess that your files might be corrupted. You can try re-cloning the repository and then move your configuration and data files over.

```sh
cd /home/silver/Playgrounds/AI
git clone https://github.com/zanllp/sd-webui-infinite-image-browsing.git IIB
cp infinite-image-browsing/iib.db IIB
cp infinite-image-browsing/.env IIB
cd IIB
python app.py
```

**Comment by 925-Studio** [comment]
> I can only guess that your files might be corrupted. You can try re-cloning the repository and then move your configuration and data files over.
> 

Appreciate your suggestion but unfortunately it didn't work. But I found something may cause the problem but I'm not an Python expertise like you do, so I'm not sure what to do with it. I found this error message when I check the 'app.py' file with VS Code. It shows on line 7.

```
...
Import "scripts.iib.tool" could not be resolved Pylance"
...
```
I've tried to select different interpreter like it suggests but none of them worked. Do you think maybe it's related with my Python environment? I'm currently using 3.10.12 on Linux and like I said earlier, never had a problem until the day before yesterday.

Funny thing is, I installed 'sd-forge' to use IIB and it works just fine. But I really don't think this is a long term solution.

**Comment by zanllp** [comment]
I found a new method. Try adding this line.

```sh
export PYTHONPATH=$PYTHONPATH:`pwd`
```


![image](https://github.com/user-attachments/assets/02c7f9ce-2831-48de-9985-4a8b2c3c7067)

**Comment by 925-Studio** [comment]
> I found a new method. Try adding this line.
> 
> ```shell
> export PYTHONPATH=$PYTHONPATH:`pwd`
> ```

Still no luck. Same error messages printed out. I can use IIB as an extension in SD-Forge, even can run it as a single page from it. Anyway, thank you for trying help me out.

**Comment by 925-Studio** [comment]
Okay, I finally got it work. Looks like my front-end experiences finally pays.

I still don't understand what causes the problem exactly (since it was working just fine), but after I changed `scripts` to `scriptss` and replaced with it all over your codes, then it works again.  My guessing is maybe something in my system conflicts with your code naming. Whatever, at least I made it work again and I can use your browser without SD-Forge.

Cheers.
